### PR TITLE
Translated API-Referenz/ReactDOM

### PR DESCRIPTION
**Pull request by do-wa**

### CODE REVIEW

File: content/docs/reference-react-dom.md [hunk 1]
```diff
@@ -111,4 +112,4 @@ When a component renders to `null` or `false`, `findDOMNode` returns `null`. Whe
 ReactDOM.createPortal(child, container)
 ```
 
-Creates a portal. Portals provide a way to [render children into a DOM node that exists outside the hierarchy of the DOM component](/docs/portals.html).
+Erstellt ein Portal. Portale sind ein Weg um [untergeordnete Elemente in einen DOM Knoten zu rendern die außerhalb der Hierarchie der DOM Komponente existieren](docs/portals.html).
```
Comment (ph1p) [inline content/docs/reference-react-dom.md]:
```suggestion
Erstellt ein Portal. Portale sind ein Weg um [untergeordnete Elemente in einen DOM-Knoten zu rendern der außerhalb der Hierarchie, der DOM-Komponente liegt](docs/portals.html).
```

File: content/docs/reference-react-dom.md [hunk 2]
```diff
@@ -6,53 +6,52 @@ category: Reference
...
+### Browser Unterstützung {#browser-support}
 
-React supports all popular browsers, including Internet Explorer 9 and above, although [some polyfills are required](/docs/javascript-environment-requirements.html) for older browsers such as IE 9 and IE 10.
+React unterstützt alle populären Browser, einschließlich Internet Explorer 9 und höher, auch wenn für ältere Browser, wie IE9 und IE10, [einige Polyfills notwendig](/docs/javascript-environment-requirements.html) sind.
 
-> Note
+> Hinweis
 >
-> We don't support older browsers that don't support ES5 methods, but you may find that your apps do work in older browsers if polyfills such as [es5-shim and es5-sham](https://github.com/es-shims/es5-shim) are included in the page. You're on your own if you choose to take this path.
+> Wir unterstützen keine älteren Browser, die keine ES5 Methoden unterstützen. Möglicherweiße funktionieren deine Applikationen auch in älteren Browser, wenn Polyfills wie z.B. [es5-shim und es5-sham](https://github.com/es-shims/es5-shim) auf deiner Seite eingebunden sind. Du bist auf dich alleine gestellt, wenn du diesen Weg einschlägst.
```
[the remark this answers is not in the document]
Comment (mxzinke) [reply content/docs/reference-react-dom.md]: #### Suggestion:
> Wir unterstützen keine älteren Browser, **die über keine ES5 Methoden verfügen.** Möglicherweiße funktionieren deine **Anwendungen** auch in älteren Browser**n**, wenn Polyfills wie z.B. [es5-shim und es5-sham](https://github.com/es-shims/es5-shim) auf deiner Seite eingebunden sind. Du bist jedoch dann auf dich alleine gestellt, wenn du diesen Weg *gehst*.

**Comment by miku86** [comment]
Hey @do-wa ,

how are you doing on your task?
Is anything holding you back?
How can we help you?

Thanks
Michael

**Comment by ph1p** [comment]
Hi @do-wa,

Please accept or decline the suggestions. So we can merge your translation.

**Comment by miku86** [comment]
@ph1p 
Any idea how to tackle this one?

**Comment by ph1p** [comment]
> @ph1p
> Any idea how to tackle this one?

The easiest way, is to copy the translated file and create a new PR.

**Comment by ph1p** [comment]
Closed by #127

**Comment by do-wa** [comment]
@ph1p 
@miku86 
I'm sorry for the problems this has created. I was honestly thinking that this Merge Request was already finished. I don't want to make any other excuses besides that this was my first Pull Request in GitHub :( 
This was my fault. Won't happen again.

Have a nice week

**Comment by ph1p** [comment]
> @ph1p
> @miku86
> I'm sorry for the problems this has created. I was honestly thinking that this Merge Request was already finished. I don't want to make any other excuses besides that this was my first Pull Request in GitHub :(
> This was my fault. Won't happen again.
> 
> Have a nice week

Oh, no problem. We just have to make sure that not too many idle PRs are created and this one is more than half a year ago now. 

And don't let it stop you from doing more PRs in the future. Just keep an eye on them ;)

We appreciate your work! Great job! Hope you can understand us .

Have a nice week, too (:

**Pull request closed without merging**